Swiftfin tvOS/Views/UserSignInView/UserSignInView.swift

```diff
@@ -49,20 +53,19 @@ struct UserSignInView: View {
     @ViewBuilder
     private var signInSection: some View {
         Section {
-            TextField(L10n.username, text: $username)
+            TextField(L10n.username, text: $username, prompt: Text("Required"))
```

Comment:
The first parameter is already the prompt, which I'm fine with just being the username label. So, the new prompt is no longer required. I don't have a design for any indicators for what fields are required except for the sign in button which _should_ have dynamic colors for whether it can be selected or not (but it just hasn't been implemented because I needed to take a break from tvOS). I'm not worried about people being confused about knowing if username is required, because that's basic requirement of the entire Jellyfin user system.

Reply:
Okay, cool. I've removed the prompt - too be honest I'm not sure why I added it. It made the field just say "Required" which is really confusing to anyone.